components/generic/CallToAction.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <SmartLink
+    :destination="url"
+    data-qa="call to action"
+    class="btn-cta btn-primary d-inline-flex align-items-center"
```

Comment:
Suggestion: 
```
class="btn btn-primary btn-cta d-inline-flex align-items-center"
```

Reply:
ok good point we can just use .btn for border-radius etc